Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN pip install pipenv
 RUN apt-get update
 # Install python dependencies in /.venv
 COPY Pipfile .
-COPY Pipfile.lock .
+#COPY Pipfile.lock .
```

Comment:
This is unexpected, usually we want the locked version of the libraries in the Docker image.